Deployment.md

```diff
@@ -0,0 +1,95 @@
+# Deployment
+
+## Deploying to an Azure Subscription for the first time
+
+If you are setting up a new micro-service pipeline from scratch, you will need to deploy all of the foundational, shared Azure resources that are necessary to support the k8 solution in Azure. All of these resources are deployed via the inception.sh script provided in this solution.
```

Comment:
"All of these resources are deployed via the inception.sh script provided in this solution."

Should we link to inception.sh?

Reply:
Good call. Updated.